website/sidebar-content-parser.ts

```
@@ -0,0 +1,95 @@
+}
+
+/**
+ * Extract navigation items from sidebar structure
+ * @param items - Sidebar items from Docusaurus
+ * @param baseUrl - Base URL for the site
+ * @param section - Section type ('docs' or 'tutorial')
+ * @returns Navigation items
+ */
+export function extractNavigationFromSidebar(
+  items: SidebarItem[],
+  baseUrl: string = 'https://podman-desktop.io',
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)